README.adoc

```diff
@@ -14,31 +14,31 @@
 // KIND, either express or implied.  See the License for the
 // specific language governing permissions and limitations
 // under the License.
+:toc:
+:toclevels: 5
+= Skupper Router
 
-= Qpid Dispatch
-
-image:https://github.com/apache/qpid-dispatch/actions/workflows/build.yaml/badge.svg[
+image:https://github.com/skupperproject/skupper-router/actions/workflows/build.yaml/badge.svg[
 "GitHub Actions"
-link="https://github.com/apache/qpid-dispatch/actions/workflows/build.yaml"]
-image:https://img.shields.io/travis/apache/qpid-dispatch.svg?logo=travisci[
+link="https://github.com/skupperproject/skupper-router/actions/workflows/build.yaml"]
+image:https://img.shields.io/travis/skupperproject/skupper-router.svg?logo=travisci[
 "Travis",
-link="https://travis-ci.com/apache/qpid-dispatch"]
-image:https://img.shields.io/github/license/apache/qpid-dispatch.svg[
+link="https://travis-ci.com/skupperproject/skupper-router"]
+image:https://img.shields.io/github/license/skupperproject/skupper-router.svg[
 "License",
-link="https://github.com/apache/qpid-dispatch/blob/main/LICENSE"]
+link="https://github.com/skupperproject/skupper-router/blob/main/LICENSE"]
+
+Skupper Router is a high-performance, lightweight AMQP 1.0 message router.
+It provides flexible and scalable interconnect between any AMQP endpoints, and is forked from the https://github.com/apache/qpid-dispatch/[qpid-dispatch] project to focus on the https://skupper.io[Skupper] use case.
 
-Qpid Dispatch is a high-performance, lightweight AMQP 1.0 message router.
-It provides flexible and scalable interconnect between any AMQP endpoints,
-whether they be clients, brokers or other AMQP-enabled services.
 
 == Building and testing
 
-NOTE: Dispatch will not build on Windows.
+NOTE: You cannot build Skupper router on Windows.
```

Comment:
```suggestion
NOTE: Skupper router only supports the Linux OS.
```

This is a change in content, but I think that's the way we should describe this. Skupper is Linux-only and it is not ashamed of it.

Reply:
I think it's a good change, and have implemented it